src/pages/basic_data/app_admin/central_import/DuplicateSearchFilterForm.tsx

```diff
@@ -0,0 +1,270 @@
+import { useTranslation } from 'react-i18next';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { Button, Checkbox, FormControlLabel, FormGroup, TextField, Typography } from '@mui/material';
+
+import { Registration } from '../../../../types/registration.types';
+import { styled as muiStyled } from '@mui/system';
+
+const StyledCenterWrapper = muiStyled('div')({
+  display: 'flex',
+  width: '100%',
+  justifyContent: 'center',
+  marginBottom: '1rem',
+  alignItems: 'center',
+});
+
+const StyledFormWrapper = muiStyled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  marginTop: '1rem',
+  marginBottom: '1rem',
+});
+
+const StyledFormElementWrapper = muiStyled('div')({
+  display: 'flex',
+  marginBottom: '1rem',
+});
+
+const StyledFormControlLabel = muiStyled(FormControlLabel)({
+  minWidth: '11rem',
+});
+
+//TODO: move out
+export interface DuplicateSearchFilters {
+  doi: string;
+  title: string;
+  author: string;
+  yearPublished: string;
+  issn: string;
+}
+
+//TODO: move out
+export const emptyDuplicateSearchFilter = {
+  doi: '',
+  title: '',
+  author: '',
+  yearPublished: '',
+  issn: '',
+};
+
+const StyledButtonWrapper = muiStyled('div')({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  marginBottom: '1rem',
+  alignItems: 'center',
+});
+
+interface DuplicateSearchFilterFormProps {
+  publication: Registration;
+  setDuplicateSearchFilters: any;
```

Comment:
Kan lage en type for SearchFilters og bruke
`SetStateAction<DuplicateSearchFilters>` eller `(filters: DuplicateSearchFilters)=>void`

Reply:
pleier å gjøre det med setstateaction så har lagt inn det nå. Typen har jeg alt.